library/src/scripts/messages/message.story.tsx

```diff
@@ -18,109 +18,144 @@ import SmartLink from "@library/routing/links/SmartLink";
 
 const story = storiesOf("Messages", module);
 
-const shortMessage = `
-Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-`;
-
+const shortMessage = `Lorem ipsum dolor sit amet, consectetur adipiscing elit.`;
 const message = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce blandit lorem ac dui porta, scelerisque placerat felis finibus.`;
-const longMessage = `
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce blandit lorem ac dui porta, scelerisque placerat felis finibus. Fusce vitae porttitor augue. Integer sagittis justo vitae nibh aliquet, a viverra ipsum laoreet. Interdum et malesuada fames ac ante ipsum primis in faucibus.
+const longMessage = `Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce blandit lorem ac dui porta, scelerisque placerat felis finibus. Fusce vitae porttitor augue. Integer sagittis justo vitae nibh aliquet, a viverra ipsum laoreet. Interdum et malesuada fames ac ante ipsum primis in faucibus.
 `;
 
 story.add("Message", () => {
     const classesMessages = messagesClasses();
     const [shortMessageFlag, setShortMessageFlag] = useState(true);
+    const [messageWithTitleFlag, setMessageWithTitleFlag] = useState(true);
     const [longMessageFlag, setLongMessageFlag] = useState(true);
-    const [fixedMessageFlag, setFixedMessageFlag] = useState(true);
-    const _shortMessage = shortMessageFlag && (
+    const [iconMessageFlag, setIconMessageFlag] = useState(true);
+    const _attachmentErrorIcon = (
```

Comment:
What do these names starting with "_" mean?

Reply:
delete this variable